test_pull_requests

```diff
@@ -23,7 +23,7 @@ require 'time'
 
 def usage
   puts <<USAGE
-Usage: test_pull_requests [--help] [--merge_pull_request <pull_id>] [--local_merge_pull_request <pull_id>] [--test_pull_request <pull_id>] [--repo <repo>]
+Usage: test_pull_requests [--help] [--merge_pull_request <pull_id>] [--local_merge_pull_request <pull_id> [--base_commit <base_commit_id>]] [--test_pull_request <pull_id>] [--repo <repo>] [--can_skip_merge_tests <pull_id>]
```

Comment:
I don't see why we need --can_skip_merge_tests

When wouldn't it be valid?  It does need an option like pretest_settings_key from the config to know what job it relies on for pretest.